presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -36,17 +38,17 @@ The ``AT TIME ZONE`` operator sets the time zone of a timestamp::
 Date and Time Functions
 -----------------------
 
-.. function:: current_date -> date
```

Comment:
Why remove the types? We should fix Sphinx so that it has a SQL language domain (instead of using the Python one) that understands the special functions which don't use parenthesis.

Reply:
Is this a real standard way of annotation a function. Removing it fixes the problem and does not loose any information since the sentence below clearly says it returns a date.